### PR TITLE
MPS: Eye op

### PR DESCRIPTION
**Pull request by kulinseth**
This can be used as a reference PR was to add Op in MPS backend.

### CODE REVIEW

File: test/test_mps.py [hunk 1]
```diff
@@ -4077,7 +4100,7 @@ def test_no_warning_on_import(self):
     def _get_not_implemented_op(self):
         # This can be changed once we actually implement `torch.eye`
         # Should return fn, args, kwargs, string_version
-        return torch.eye, (2,), {"device": "mps"}, "torch.eye(2, device='mps')"
+        return torch.bincount, (torch.tensor([4, 3, 6, 3, 4], device='mps')), {}, "torch.bincount(torch.tensor([4, 3, 6, 3, 4], device='mps'))"
```
Comment (albanD) [inline test/test_mps.py:4103]: nit: update the comment above to say `bincount`.

**Review by albanD — APPROVED** [review]
That sounds quite good.
We should mention in the PR description though that this is a temporary way to do testing until MPS gets integrated with the more generic testing we have.

File: test/test_mps.py (hunk 1 above)
Comment (kulinseth) [reply test/test_mps.py:4103]: Done.

**Comment by kulinseth** [comment]
The build failures seem un-related to this checkin:

```
Prepare all required actions
[2](https://github.com/pytorch/pytorch/runs/6620803046?check_suite_focus=true#step:11:2)
Getting action download info
[3](https://github.com/pytorch/pytorch/runs/6620803046?check_suite_focus=true#step:11:3)
Download action repository 'actions/upload-artifact@v2' (SHA:82c141cc518b40d92cc801eee768e7aafc9c2fa2)
[4](https://github.com/pytorch/pytorch/runs/6620803046?check_suite_focus=true#step:11:4)
Warning: Failed to download action 'https://api.github.com/repos/actions/upload-artifact/tarball/82c141cc518b40d92cc801eee768e7aafc9c2fa2'. Error: Response status code does not indicate success: 401 (Unauthorized).
[5](https://github.com/pytorch/pytorch/runs/6620803046?check_suite_focus=true#step:11:5)
Warning: Back off 19.944 seconds before retry.
[6](https://github.com/pytorch/pytorch/runs/6620803046?check_suite_focus=true#step:11:6)
Warning: Failed to download action 'https://api.github.com/repos/actions/upload-artifact/tarball/82c141cc518b40d92cc801eee768e7aafc9c2fa2'. Error: Response status code does not indicate success: 401 (Unauthorized).
[7](https://github.com/pytorch/pytorch/runs/6620803046?check_suite_focus=true#step:11:7)
Warning: Back off 12.776 seconds before retry.
[8](https://github.com/pytorch/pytorch/runs/6620803046?check_suite_focus=true#step:11:8)
Error: Response status code does not indicate success: 401 (Unauthorized).
```

can we merge this @albanD

**Review by malfet — APPROVED** [review]

**Pull request closed without merging**